policyengine_us/parameters/gov/states/co/ccap/base_parent_fee/first_and_second_multiplication_factor.yaml

```
@@ -1,12 +1,21 @@
-description: Colorado Child Care Assistance Program multiplies adjusted gross income or federal poverty guidelines with this factor to calculate base parent fee.
+description: Colorado Child Care Assistance Program multiplies adjusted gross income by the following factor to calculate base parent fee, based on gross income relative to the federal poverty guidelines.  
```

Please also change other variables (Python and PolicyEngine) and descriptions/labels to refer consistently to countable income or adjusted gross income (appears to be the former).
```suggestion
description: Colorado calculates its Child Care Assistance Program base parent fee by applying this rate schedule to the ratio of countable income to the federal poverty guideline, then multiplying the result by the federal poverty guideline.
```